custom/icds_reports/static/js/directives/access-denied/access-denied.directive.js

```diff
@@ -1,9 +1,6 @@
-
-var url = hqImport('hqwebapp/js/initial_page_data').reverse;
-
-window.angular.module('icdsApp').directive('accessDenied', function () {
-    return {
-        restrict: 'E',
-        templateUrl: url('icds-ng-template', 'access-denied.directive'),
-    };
+window.angular.module('icdsApp').component('accessDenied', {
+    templateUrl: function () {
+        var url = hqImport('hqwebapp/js/initial_page_data').reverse;
```

Comment:
was there a reason for shifting the `url` to the local scope?

Reply:
Yeah. One of the reasons was lint warnings of giving this variable redundant global scope. (I dont remember the other reason exactly :expressionless: )